examples/messengers/telegram/4_conditions.py

```diff
@@ -0,0 +1,119 @@
+# %% [markdown]
+"""
+# 4. Conditions
+
+This example shows how to process Telegram updates in your script
+and reuse handler triggers from the `pytelegrambotapi` library.
+"""
+
+# %%
+import os
+
+from dff.script import TRANSITIONS, RESPONSE
+
+from dff.messengers.telegram import (
+    PollingTelegramInterface,
+    TelegramMessenger,
+    message_handler,
+    handler,
+    UpdateType,
+)
+from dff.pipeline import Pipeline
+from dff.messengers.telegram import TelegramMessage
+from dff.utils.testing.common import is_interactive_mode
+
+
+# %% [markdown]
+"""
+In our Telegram module, we adopted the system of filters
+available in the `pytelegrambotapi` library.
+
+You can use `message_handler` to filter text messages from telegram in various ways.
+Filling the `command` argument will cause the handler to only react to listed commands.
+`func` argument on the other hand allows you to define arbitrary conditions.
+`regexp` creates a regular expression filter, etc.
+
+Note:
+Using `message_handler(...)` is the same as using `handler(target_type=UpdateTypes.MESSAGE, ...)`.
+
+Note:
+It is possible to use `cnd.exact_match` as a condition (as seen in previous examples).
+However, the functionality of that approach is lacking:
+At this moment only two fields of `Message` are set during update processing:
+
+- `text` stores a `text` field of `message` updates
+- `commands` stores a `data` field of `callback_query` updates
+
+For more information see example `3_buttons_with_callback.py`.
+"""
+
+
+# %%
+# Like Telebot, TelegramMessenger only requires a token to run.
+# However, all parameters from the Telebot class can be passed as keyword arguments.
+messenger = TelegramMessenger(os.getenv("TG_BOT_TOKEN", "SOMETOKEN"))
+
+
+# %%
+script = {
+    "greeting_flow": {
+        "start_node": {
+            RESPONSE: TelegramMessage(text="Hi"),
+            TRANSITIONS: {"node1": message_handler(commands=["start", "restart"])},
+        },
+        "node1": {
+            RESPONSE: TelegramMessage(text="Hi, how are you?"),
+            TRANSITIONS: {"node2": handler(target_type=UpdateType.MESSAGE, regexp="fine")},
+        },
+        "node2": {
+            RESPONSE: TelegramMessage(text="Good. What do you want to talk about?"),
+            TRANSITIONS: {"node3": message_handler(func=lambda msg: "music" in msg.text)},
+        },
+        "node3": {
+            RESPONSE: TelegramMessage(text="Sorry, I can not talk about music now."),
+            TRANSITIONS: {"node4": message_handler(func=lambda msg: True)},
+        },
+        "node4": {
+            RESPONSE: TelegramMessage(text="bye"),
+            TRANSITIONS: {"node1": message_handler(commands=["start", "restart"])},
+        },
+        "fallback_node": {
+            RESPONSE: TelegramMessage(text="Ooops"),
+            TRANSITIONS: {"node1": message_handler(commands=["start", "restart"])},
```

Comment:
`message_handle` looks as not related one with conditions. It should be good to refactor the name of  `message_handle`

Reply:
https://github.com/deeppavlov/dialog_flow_framework/pull/13/commits/a17caa673b362b0c30b5c2cb73c25c6282b7a421